artifacts/vulnerabilitymanifestsummaries/01-example.yaml

```diff
@@ -1,19 +1,48 @@
 apiVersion: spdx.softwarecomposition.kubescape.io/v1beta1
 kind: VulnerabilityManifestSummary
 metadata:
-  name: "some-vm-summary"
-  namespace: kubescape
+  annotations:
+    kubescape.io/wlid: wlid://cluster-gke_armo-test-clusters_us-central1-c_dwertent-large-node/namespace-kubescape/deployment-storage
+    kubescape.io/image-id: quay.io/kubescape/storage@sha256:12cb1a854e5c287aeb9f227b70b16c35a0dd1981ea1f2b8aea1c1176ea0ba4eb
+    kubescape.io/workload-container-name: apiserver
+    kubescape.io/status: ""
   labels:
-    sample-label: "true"
+    kubescape.io/workload-api-group: apps
+    kubescape.io/workload-api-version: v1
+    kubescape.io/workload-namespace: kubescape
+    kubescape.io/workload-kind: Deployment
+    kubescape.io/workload-name: storage
+    kubescape.io/workload-container-name: apiserver
+    kubescape.io/context: filtered # if relevancy is available 
+  name: kubescape-deployment-storage-apiserver # <namespace>-<kind>-<name>-<container name>
+  namespace: kubescape
 spec:
   severities:
     critical:
-      all: 5
+      all: 0
+      relevant: 0
     high:
       all: 10
-    medium:
-      all: 15
+      relevant: 4
     low:
-      all: 20
+      all: 0
+      relevant: 0
+    medium:
+      all: 2
+      relevant: 1
+    negligible:
+      all: 0
+      relevant: 0
     unknown:
-      all: 5
+      all: 0
+      relevant: 0
+  vulnerabilities:
+    basic:
+      namespace: "kubescape"
+      name: "gke.gcr.io-fluent-bit-gke-exporter-sha256-93014f5d546376de76c21f48bf30a6d1df3db4a413a1c3009c59fe46fa83eee8-83eee8"
+      kind: "vulnerabilitymanifests"
+    filtered: 
+      namespace: "kubescape"
+      name: "kubescape-replicaset-storage-cf988cc64-cd47-4240"
+      kind: "vulnerabilitymanifests"
+      
```

Comment:
style: missing proper trailing newline

This is probably because your editor saves newlines as CRLF. Try using LF.